core/src/js/utilities/events.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Create an event with the specified name in a browser-agnostic way.
+ * @param {string} eventName - the name of the event
+ * @return {Event} - instance of event which can be dispatched / listened for
+ */
+export const createEvent = (eventName = '') => {
+  if (typeof eventName !== 'string' || eventName.length <= 0) {
+    return null;
```

Comment:
Do we want this to quietly fail or do we want to throw an exception here?

Reply:
Normally I like to defend against errors, but in this case I guess a compile time error is better than a runtime error. I'll remove the default.